src/shared/DMIParser.c

```diff
@@ -0,0 +1,100 @@
+/*
+ * Created by Aavild on 19/11/2021.
```

Comment:
Do we want file headers for every file? In that case we should create a standard for it to formalise the headers. However imo we dont want the trouble of doing this.

An actual file header might look like this

```c
/* Written under the MIT license. 
 * [Insert license here]
 * Original author:
 * Modifications:
 *     - [date] [change]
 *     - [date] [change]
 *     - [date] [change]
 */
```
In my opinion this is too much work and not necessary for a project of this size.

tl;dr: Remove this comment 😀

Reply:
resolved